AirCasting/Utils/Keychain/KeychainManager.swift

```diff
@@ -0,0 +1,51 @@
+// Created by Lunar on 16/08/2022.
+//
+
+import Foundation
+
+class KeychainManager {
+    enum KeychainError: Error {
+        case duplicateEntry
+        case unknown(OSStatus)
+    }
+    
+    static func save(service: String, account: String, password: Data) throws {
+        let query: [String: AnyObject] = [
+            kSecClass as String: kSecClassGenericPassword,
+            kSecAttrService as String: service as AnyObject,
+            kSecAttrAccount as String: account as AnyObject,
+            kSecValueData as String: password as AnyObject,
+        ]
+        
+        let status = SecItemAdd(query as CFDictionary, nil)
+        
+        guard status != errSecDuplicateItem else {
+            throw KeychainError.duplicateEntry
+        }
+        
+        guard status != errSecSuccess else {
+            throw KeychainError.unknown(status)
+        }
+        
+        Log.info("Saved password to keychain")
+    }
+    
+    static func get(service: String, account: String) -> Data? {
```

Comment:
the other function is throwing, this one is silent, maybe we should unify? (I'm saying let's go with throwing)